kickoff/views/submit.py

```diff
@@ -8,15 +8,38 @@
 from kickoff.model import getReleaseTable
 from kickoff.views.forms import FennecReleaseForm, FirefoxReleaseForm, \
     ThunderbirdReleaseForm, getReleaseForm
+import urllib2
+import json
 
 log = logging.getLogger(__name__)
 
 
+def retrieveADI():
+    defaultURL = "http://localhost/"
+    # Download the source of the data
+    # Download it from a remote interface to avoid a dependency on vertica db
+    # We do that in Python to avoid the Cross Origin complexity in JS
+    try:
+        response = urllib2.urlopen(defaultURL + 'partial-beta.json')
+        jsonSourceBeta = response.read().strip()
+        response = urllib2.urlopen(defaultURL + 'partial-release.json')
+        jsonSourceRelease = response.read().strip()
+        response = urllib2.urlopen(defaultURL + 'partial-esr.json')
+        jsonSourceESR = response.read().strip()
+        return jsonSourceBeta, jsonSourceRelease, jsonSourceESR
+    except urllib2.HTTPError, e:
```

Comment:
I thought we were doing this on in the frontend based on the e-mail thread. The thing I don't like about doing it on the server is that ship-it can become unusable if the connection to the server that hosts this file hangs.